src/components/Items/ProfileContact.js

```diff
@@ -82,7 +87,8 @@ ProfileContact.propTypes = {
   showArrow: PropTypes.bool,
   userId: PropTypes.string,
   ImageComponent: PropTypes.node,
-  from: PropTypes.string
+  from: PropTypes.string,
+  isYou: PropTypes.bool
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isYou` prop has been added to PropTypes validation. This is a good practice to ensure that the correct type of prop is passed to the component.

```diff
-  from: PropTypes.string
+  from: PropTypes.string,
+  isYou: PropTypes.bool
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->